README.md

```diff
@@ -1,17 +1,120 @@
+`className`: Optional. Additional classes to customize the container's styles.
+`children`: Required. The content to be wrapped inside the container.
+
+---
+
+#### Button: 
+This element is implemented using the **Button** component. It can be used to create buttons with consistent styling across the application.
+
+```jsx
+import Button from "../components/Button"; 
+
+<Button>Sample Button</Button>
+```
+**Props**:
+`type`: Optional. Specifies the type of the button (e.g., "submit", "reset", "button").
+`children`: Required. The content to be displayed inside the button.
+`className`: Optional. A string of additional CSS classes to apply.
+`...props`: Optional. Any other valid attributes or event handlers.
+
+---
+
+#### Link: 
+
+This element is implemented using the **BaseLink** component. It creates consistent links throughout the application.
+
+```jsx
+import BaseLink from "../components/BaseLink";
+
+<BaseLink to="/">Sample Link</BaseLink>
+```
+
+**Props**:
+`to`: Required. The destination URL for the link.
+`children`: Required. The content to be displayed inside the link.
+`className`: Optional. A string of additional CSS classes to apply.
+
+---
+
+#### Input: 
+
+This element is implemented using the **InputField** component. Note: This component is specifically designed for input fields where the type attribute is used for textual data entry (e.g., "text", "email", "password"). It is not suitable for input types such as "button", "checkbox", etc.
+
+
+
+```jsx
+import InputField from "../components/InputField";
+              
+<InputField label="Email" id="email" type="email" placeholder="Placeholder" />
+```
+
+**Props**:
+`label`: Required. The label text for the input field.
+`id`: Required. The id attribute for the input element.
+`type`: Optional. The type of the input (e.g., "text", "email", "password").
+`placeholder`: Optional. The placeholder text for the input field.
+`hint`: Optional. A hint text displayed below the input field.
+`className`: Optional. A string of additional CSS classes to apply.
+
+---
+
+#### Alert: 
+
+This element is implemented using the **Alert** component. It displays alerts with support for an optional error type. If no type is specified, the alert defaults to success.
+
+```jsx
+import Alert from "../components/Alert";
+              
+<Alert title="Success">Sample success message</Alert>
+<Alert type="error" title="Error">Sample error message</Alert>
+```
+
+**Props**:
+`type`: Optional. Specifies the type of alert. If set to error, the alert will be styled accordingly. If omitted, the alert defaults to success.
+`title`: Optional. The title of the alert.
+`children`: Optional. The content to be displayed inside the alert.
+
+---
+
+## Setting up local development environment
 
 1. Clone this repository to the folder that was already created for both the front-end and back-end repos
 2. Run `npm install` to install dependencies
```

Comment:
A few suggested changes to below
- Update item 5 to the correct port (http://localhost:5173) and remove the language around seeing data from the back-end 
- Remove item 6
- Consider removing the sections below since they were tips for the start of practicum (and you can delete any associated images from the repo as well)

Reply:
Done!